packages/snjs/test/lib/settings.test.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Platform,
+  Environment,
+  Uuid,
+  DeinitSource,
+  SNApplication,
+} from '@Lib/index';
+import { createInitAppWithRandNamespace } from '../factory';
+
+describe('settings', function () {
+  const fakeSetting: any = 'FAKE_SETTING';
+  const fakePayload = 'Im so meta even this acronym';
+  const updatedFakePayload = 'is meta';
+
+  let snApp: SNApplication;
+
+  beforeEach(async function () {
+    snApp = await createInitAppWithRandNamespace(
+      Environment.Web,
+      Platform.MacWeb
+    );
+    const email = Uuid.GenerateUuidSynchronously();
+    const password = Uuid.GenerateUuidSynchronously();
+    const ephemeral = false;
+    const mergeLocal = true;
+
+    await snApp.register(email, password, ephemeral, mergeLocal);
+
+    // We have to wait for sync to finish, otherwise
+    // the SNSyncService.handleSuccessServerResponse is fired
+    // after the test has finished causing an error
+    await snApp.sync();
```

Comment:
This isn't the case with other test suites. Shouldn't be necessary here.

Reply:
(Likely a sign that something is incorrect)

Reply:
It was a remnant of my attempt to modify the old testing infrastructure for this purpose. There's no need now for that. I think it was the Factory.safeDeinit that solved it.